src/routes/middleware/validators/user.validator.ts

```diff
@@ -9,10 +9,18 @@ const password = body("password")
     "Password must have minimum length of 8 and contain at least 1 lowercase, 1 uppercase character, 1 symbol and 1 number"
   );
 
+const id = param("id")
+  .isMongoId()
+  .withMessage("Id must be valid MongoDB id")
```

Comment:
You may not want to expose details about app internal structure for users. This can be helpful for potential attacker.